os/mm/mm_heap/mm_heapinfo_parse_heap.c

```diff
@@ -110,6 +115,17 @@ void heapinfo_parse_heap(FAR struct mm_heap_s *heap, int mode, pid_t pid)
 
 	ASSERT(mm_check_heap_corruption(heap) == OK);
 
+#ifdef CONFIG_WATCHDOG
+	int wd_fd = open(CONFIG_WATCHDOG_DEVPATH, O_RDONLY);
+#ifndef CONFIG_MM_ASSERT_ON_FAIL
+	if (wd_fd < 0) {
+		mfdbg("Fail to open %s, errno %d\n", CONFIG_WATCHDOG_DEVPATH, get_errno());
+		mfdbg("The heap usage dump doesn't printed\n");
```

Comment:
Let's change the sentence as below
`It will not print detail of heap usages to prevent watchdog reset.`